### PR TITLE
ROC-3374: page title money claims

### PR DESCRIPTION
**Pull request by rattur**
### JIRA link

https://tools.hmcts.net/jira/browse/ROC-3374

### Change description

Initial page title changes for claim and response journey

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ x] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ x] No

### CODE REVIEW

File: src/main/features/claim/views/amount-exceeded.njk [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {% extends "layout.njk" %}
 
-{% set title = t('Excess Money claim') %}
-{% set heading = t('Make a money claim') %}
+{% set heading = 'Make a money claim' %}
```
Comment (timja-kainos) [inline src/main/features/claim/views/amount-exceeded.njk]: This heading is incorrect. It should reflect the content on the page

File: src/main/views/govukTemplate.njk [hunk 2]
```diff
@@ -4,7 +4,11 @@
 <!--[if gt IE 8]><!--><html lang="{{ html_lang|default('en') }}"><!--<![endif]-->
 <head>
   <meta charset="utf-8" />
-  <title>{{ t(title) }}</title>
+  {% if heading %}
+    <title>{{ t(heading) }} - {{ t(moneyClaims) }}</title>
+  {% else %}
+    <title>{{ t(moneyClaims) }}</title>
```
Comment (timja-kainos) [inline src/main/views/govukTemplate.njk]: why would you not have a heading here? moneyclaims probably isn't a great fallback

File: src/main/views/layout.njk [hunk 3]
```diff
@@ -25,3 +25,5 @@
   {% include "includes/scripts.njk" %}
   {% block additional_scripts %}{% endblock %}
 {% endblock %}
+
```
Comment (timja-kainos) [inline src/main/views/layout.njk]: why add these lines?

File: src/test/features/claim/routes/confirmation.ts [hunk 4]
```diff
@@ -42,7 +42,7 @@ describe('Claim issue: confirmation page', () => {
         await request(app)
           .get(ClaimPaths.confirmationPage.evaluateUri({ externalId: externalId }))
           .set('Cookie', `${cookieName}=ABC`)
-          .expect(res => expect(res).to.be.successful.withText('Claim confirmation'))
+          .expect(res => expect(res).to.be.successful.withText('Claim submitted'))
```
Comment (timja-kainos) [inline src/test/features/claim/routes/confirmation.ts]: why has this changed?

Comment (rattur) [reply src/test/features/claim/routes/confirmation.ts]: Test case failed as it was looking for title 'Claim confirmation'

File: src/main/views/govukTemplate.njk (hunk 2 above)
Comment (rattur) [reply src/main/views/govukTemplate.njk]: It seems that when trying to set the heading on any confirmation page, the heading does not seem to render within the 'govuk-box-highlight' css

File: src/main/views/layout.njk (hunk 3 above)
Comment (rattur) [reply src/main/views/layout.njk]: Was trying a solution within this file originally so will remove line left behind

File: src/main/views/govukTemplate.njk (hunk 2 above)
Comment (timja-kainos) [reply src/main/views/govukTemplate.njk]: this page should have the correct title though this is a hack which breaks it, i would suggest a special variable instead, confirmation_heading maybe?

Comment (rattur) [reply src/main/views/govukTemplate.njk]: Sounds good

File: src/main/modules/nunjucks/index.ts [hunk 5]
```diff
@@ -126,6 +126,8 @@ export class Nunjucks {
     nunjucksEnv.addGlobal('HowMuchPaidClaimantOption', HowMuchPaidClaimantOption)
     nunjucksEnv.addGlobal('Service', Service)
     nunjucksEnv.addGlobal('cookieText', `GOV.UK uses cookies make the site simpler. <a href="${AppPaths.cookiesPage.uri}">Find out more about cookies</a>`)
+    nunjucksEnv.addGlobal('moneyClaims', `Money Claims`)
+    nunjucksEnv.addGlobal('headingVisible', `true`)
```
Comment (robertparkinson) [inline src/main/modules/nunjucks/index.ts]: the true shouldn't be wrapped in quotes

Comment (rattur) [reply src/main/modules/nunjucks/index.ts]: But the addGlobal function expects two strings

Comment (robertparkinson) [reply src/main/modules/nunjucks/index.ts]: according to the method the second option is of type any: 
```addGlobal(name: string, value: any)```
and using true is more accurate from a type point of view

Comment (rattur) [reply src/main/modules/nunjucks/index.ts]: Ok, I'll change and see if the application still runs ok. I thought it was two strings but yes your correct its a string and any

File: src/main/views/govukTemplate.njk [hunk 6]
```diff
@@ -4,7 +4,9 @@
 <!--[if gt IE 8]><!--><html lang="{{ html_lang|default('en') }}"><!--<![endif]-->
 <head>
   <meta charset="utf-8" />
-  <title>{{ t(title) }}</title>
+  {% if heading %}
```
Comment (sabahirfan) [inline src/main/views/govukTemplate.njk]: do we need this condition?

Comment (rattur) [reply src/main/views/govukTemplate.njk]: Your probably right, now that we have a 'headingVisible' condition variable. Will check the templates throughout the application to verify

File: src/main/modules/nunjucks/index.ts [hunk 7]
```diff
@@ -126,6 +126,8 @@ export class Nunjucks {
     nunjucksEnv.addGlobal('HowMuchPaidClaimantOption', HowMuchPaidClaimantOption)
     nunjucksEnv.addGlobal('Service', Service)
     nunjucksEnv.addGlobal('cookieText', `GOV.UK uses cookies make the site simpler. <a href="${AppPaths.cookiesPage.uri}">Find out more about cookies</a>`)
+    nunjucksEnv.addGlobal('moneyClaims', `Money Claims`)
```
Comment (geek-soft) [inline src/main/modules/nunjucks/index.ts]: Please rename this to be `serviceName` or similar. A `moneyClaims` = `Money Claims` does not look good.

File: src/main/features/response/views/incomplete-submission.njk [hunk 8]
```diff
@@ -1,14 +1,12 @@
...
 
-{% set title = t('Incomplete submission') %}
-
 {% block content %}
 
   {{
     remainingTasksFragment(
-      heading = 'You need to complete all sections before you submit your response',
       tasks = tasks,
+      heading = 'You need to complete all sections before you submit your response',
```
Comment (geek-soft) [inline src/main/features/response/views/incomplete-submission.njk]: Please add heading to the page. Title for the page is ` - Money Claims`

File: src/main/features/claim/views/incomplete-submission.njk [hunk 9]
```diff
@@ -1,8 +1,6 @@
 {% extends "layout.njk" %}
 {% from "remainingTasksFragment.njk" import remainingTasksFragment  %}
 
-{% set title = t('Incomplete submission') %}
```
Comment (geek-soft) [inline src/main/features/claim/views/incomplete-submission.njk]: Please set heading for that page.

File: src/main/features/response/views/task-list.njk [hunk 10]
```diff
@@ -1,15 +1,11 @@
 {% extends "layout.njk" %}
 {% from "task-list.njk" import task, taskListHeading, disabledTask %}
 {% from "timeRemaining.njk" import timeRemaining %}
-
-{% set title = t('Task list | Money claim') %}
+{% set heading = 'Respond to a money claim' %}
```
Comment (geek-soft) [inline src/main/features/response/views/task-list.njk]: Please add blank line between

**Comment by pragnesh-patel** [comment]
can we get another dev to check this PR? @singh-surjeet  @rattur @sabahirfan @jacmurphy please

File: src/test/a11y/a11y.ts [hunk 11]
```diff
@@ -68,7 +67,7 @@ function ensureHeadingIsIncludedInPageTitle (text: string): void {
     expect(title).to.be.equal(`${heading[1]} - Money Claims`)
   } else {
     expect(title).to.be.not.equal(' - Money Claims')
-    logger.warn(`No heading found on page titled '${title}' exists`)
+    console.log(`NOTE: No heading found on page titled '${title}' exists`)
```
Comment (yarpo) [inline src/test/a11y/a11y.ts]: do we want to use `console.log`?

Comment (geek-soft) [reply src/test/a11y/a11y.ts]: Yes, it gives me better results then when I used logger. I am OK with console.log to print output in tests.

**Comment by geek-soft** [comment]
```
  Accessibility
    Page /eligibility
      ✓ should be accessible (2768ms)
    Page /eligibility/claim-value
      ✓ should be accessible (1417ms)
    Page /eligibility/help-with-fees
      ✓ should be accessible (1449ms)
    Page /eligibility/single-defendant
      ✓ should be accessible (1528ms)
    Page /eligibility/claim-type
      ✓ should be accessible (1648ms)
    Page /eligibility/claimant-address
      ✓ should be accessible (1560ms)
    Page /eligibility/defendant-address
      ✓ should be accessible (1265ms)
    Page /eligibility/over-18
      ✓ should be accessible (1397ms)
    Page /eligibility/defendant-age
      ✓ should be accessible (1290ms)
    Page /eligibility/government-department
      ✓ should be accessible (1224ms)
    Page /eligibility/claim-is-for-tenancy-deposit
      ✓ should be accessible (1256ms)
    Page /eligibility/eligible
      ✓ should be accessible (1295ms)
    Page /eligibility/not-eligible
      ✓ should be accessible (1813ms)
    Page /eligibility/mcol-eligibility
      ✓ should be accessible (1222ms)
    Page /claim/task-list
      ✓ should be accessible (1451ms)
    Page /claim/resolving-this-dispute
      ✓ should be accessible (1355ms)
    Page /claim/completing-claim
      ✓ should be accessible (1297ms)
    Page /claim/claimant-individual-details
      ✓ should be accessible (1323ms)
    Page /claim/claimant-company-details
      ✓ should be accessible (1316ms)
    Page /claim/claimant-sole-trader-details
      ✓ should be accessible (1409ms)
    Page /claim/claimant-organisation-details
      ✓ should be accessible (1482ms)
    Page /claim/defendant-individual-details
      ✓ should be accessible (1336ms)
    Page /claim/defendant-company-details
      ✓ should be accessible (1389ms)
    Page /claim/defendant-sole-trader-details
      ✓ should be accessible (1270ms)
    Page /claim/defendant-organisation-details
      ✓ should be accessible (1293ms)
    Page /claim/claimant-dob
      ✓ should be accessible (1251ms)
    Page /claim/claimant-mobile
      ✓ should be accessible (1362ms)
    Page /claim/defendant-email
      ✓ should be accessible (1399ms)
    Page /claim/amount
      ✓ should be accessible (1427ms)
    Page /claim/interest
      ✓ should be accessible (1293ms)
    Page /claim/interest-type
      ✓ should be accessible (1320ms)
    Page /claim/interest-rate
      ✓ should be accessible (1304ms)
    Page /claim/interest-date
      ✓ should be accessible (1352ms)
    Page /claim/interest-start-date
      ✓ should be accessible (1339ms)
    Page /claim/interest-end-date
      ✓ should be accessible (1283ms)
    Page /claim/interest-total
      ✓ should be accessible (1255ms)
    Page /claim/interest-continue-claiming
      ✓ should be accessible (1373ms)
    Page /claim/interest-how-much
      ✓ should be accessible (2489ms)
    Page /claim/total
      ✓ should be accessible (3320ms)
    Page /claim/reason
      ✓ should be accessible (1349ms)
    Page /claim/timeline
      ✓ should be accessible (1413ms)
    Page /claim/evidence
      ✓ should be accessible (1815ms)
    Page /claim/check-and-send
      ✓ should be accessible (1663ms)
    Page /claim/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/confirmation
NOTE: No heading found on page titled 'Confirmation - Money Claims' exists
      ✓ should be accessible (1513ms)
    Page /claim/claimant-party-type-selection
      ✓ should be accessible (1584ms)
    Page /claim/defendant-party-type-selection
      ✓ should be accessible (1468ms)
    Page /claim/incomplete-submission
      ✓ should be accessible (1931ms)
    Page /claim/amount-exceeded
      ✓ should be accessible (1296ms)
    Page /first-contact/start
      ✓ should be accessible (1854ms)
    Page /first-contact/claim-reference
      ✓ should be accessible (1554ms)
    Page /first-contact/claim-summary
      ✓ should be accessible (1719ms)
    Page /first-contact/access-denied
      ✓ should be accessible (6029ms)
    Page /first-contact/claimant-has-requested-ccj
      ✓ should be accessible (3175ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/task-list
      ✓ should be accessible (1493ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/your-details
      ✓ should be accessible (1710ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/your-dob
      ✓ should be accessible (1373ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/your-mobile
      ✓ should be accessible (1366ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/more-time-request
      ✓ should be accessible (1438ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/more-time-confirmation
      ✓ should be accessible (1450ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/response-type
      ✓ should be accessible (1474ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/your-defence
      ✓ should be accessible (1336ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/free-mediation
      ✓ should be accessible (1378ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/check-and-send
      ✓ should be accessible (1483ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/confirmation
NOTE: No heading found on page titled 'Confirmation - Money Claims' exists
      ✓ should be accessible (1336ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/counter-claim
      ✓ should be accessible (1370ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/incomplete-submission
      ✓ should be accessible (1422ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/timeline
      ✓ should be accessible (1309ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/evidence
      ✓ should be accessible (1385ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/eligibility/send-your-response-by-email
      ✓ should be accessible (1291ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/when-did-you-pay
      ✓ should be accessible (1326ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/eligibility/how-much-paid-claimant
      ✓ should be accessible (1385ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/how-much-paid
      ✓ should be accessible (1283ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/how-much-owed
      ✓ should be accessible (1382ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/reject-all-of-claim
      ✓ should be accessible (1553ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/reject-part-of-claim
      ✓ should be accessible (1435ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/when-will-you-pay
      ✓ should be accessible (1386ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/your-payment-plan
      ✓ should be accessible (1280ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/impact-of-dispute
      ✓ should be accessible (1366ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/claim-details
      ✓ should be accessible (1529ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/summary
      ✓ should be accessible (1517ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/ccj/date-of-birth
      ✓ should be accessible (2181ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/ccj/paid-amount
      ✓ should be accessible (1849ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/ccj/paid-amount-summary
      ✓ should be accessible (1986ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/ccj/payment-options
      ✓ should be accessible (1849ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/ccj/check-and-send
      ✓ should be accessible (1743ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/ccj/pay-by-set-date
      ✓ should be accessible (1709ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/ccj/repayment-plan
      ✓ should be accessible (1881ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/ccj/confirmation
NOTE: No heading found on page titled 'Confirmation - Money Claims' exists
      ✓ should be accessible (1473ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/offer/your-offer
      ✓ should be accessible (1315ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/offer/offer-confirmation
NOTE: No heading found on page titled 'Confirmation - Money Claims' exists
      ✓ should be accessible (1384ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/offer/settle-out-of-court
      ✓ should be accessible (1855ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/offer/response
NOTE: No heading found on page titled 'Offer - Money Claims' exists
      ✓ should be accessible (1516ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/offer/make-agreement
      ✓ should be accessible (1444ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/offer/countersign-agreement
      ✓ should be accessible (1244ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/offer/declaration
      ✓ should be accessible (1553ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/offer/settled
NOTE: No heading found on page titled 'Confirmation - Money Claims' exists
      ✓ should be accessible (1362ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/offer/accepted
NOTE: No heading found on page titled 'Confirmation - Money Claims' exists
      ✓ should be accessible (1252ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/offer/rejected
NOTE: No heading found on page titled 'Confirmation - Money Claims' exists
      ✓ should be accessible (1351ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/statement-of-means/start
      ✓ should be accessible (1283ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/statement-of-means/what-you-need
      ✓ should be accessible (1466ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/statement-of-means/residence
      ✓ should be accessible (1360ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/statement-of-means/dependants
      ✓ should be accessible (1323ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/statement-of-means/education
      ✓ should be accessible (1426ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/statement-of-means/maintenance
      ✓ should be accessible (1277ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/statement-of-means/supported-by-you
      ✓ should be accessible (1371ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/statement-of-means/employment
      ✓ should be accessible (1299ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/statement-of-means/employers
      ✓ should be accessible (1320ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/statement-of-means/self-employed
      ✓ should be accessible (1343ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/statement-of-means/unemployed
      ✓ should be accessible (1311ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/statement-of-means/bank-accounts
      ✓ should be accessible (1717ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/statement-of-means/debts
      ✓ should be accessible (2954ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/statement-of-means/monthly-income
      ✓ should be accessible (1867ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/statement-of-means/monthly-expenses
      ✓ should be accessible (2101ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/statement-of-means/court-orders
      ✓ should be accessible (1281ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/pay-by-set-date/payment-date
      ✓ should be accessible (1322ms)
    Page /case/91e1c70f-7d2c-4c1e-a88f-cbb02c0e64d6/response/pay-by-set-date/explanation
      ✓ should be accessible (1330ms)


  116 passing (3m)
```

**Pull request merged**